TrueCraft.API/Ray.cs

```diff
@@ -5,12 +5,24 @@
 
 namespace TrueCraft.API
 {
-    // Mostly taken from the MonoXna project, which is licensed under the MIT license
```

Comment:
That comment was not intended to be exported as part of the public API documentation.